doc/source/cluster/cloud.rst

```diff
@@ -327,7 +325,7 @@ you can use a tool such as ``nmap`` or ``nc``.
   Host is up, received echo-reply ttl 60 (0.00087s latency).
   rDNS record for 123.456.78.910: compute04.berkeley.edu
   PORT     STATE SERVICE REASON         VERSION
-  6379/tcp open  redis   syn-ack ttl 60 Redis key-value store
+  6379/tcp open  redis?  syn-ack
```

Comment:
Not sure about this one, @ericl  is this correct?

Reply:
This is what I got from running the command. I believe because of the `6379` port, `nm` guessed `redis` as the service. But because of not using redis protocol, it is suffixed with `?`.